plugins/modules/gitlab_issue.py

```diff
@@ -183,7 +183,7 @@ def get_milestone(self, milestone_id, group):
     def get_issue(self, title, state_filter):
         issues = []
         try:
-            issues = self.project.issues.list(title=title, state=state_filter)
+            issues = self.project.issues.list(search=title, state=state_filter)
```

Comment:
As discussed - you can pass any filter currently available in the GitLab API, and in this case it's `search` for the V4 of GitLab's API. But it's true you'll want to avoid the collision with python's `in` causing a syntax error. Something like this:

```suggestion
            issues = self.project.issues.list(query_parameters={"search": title, "in": "title", "state": state_filter} )
```